packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx

```diff
@@ -127,6 +127,12 @@ export class InputMapFindPlace<CustomSurvey, CustomHousehold, CustomHome, Custom
     }
 
     onValueChange = (feature?: GeoJSON.Feature<GeoJSON.Point, FeatureGeocodedProperties>) => {
+        if (feature) {
+            feature.geometry.coordinates = feature.geometry.coordinates.map((coordinate) => {
+                return Number(coordinate.toFixed(this.props.widgetConfig.coordinatesPrecision ?? 5));
```

Comment:
Is `??` like `||` ?

Reply:
Almost!
See https://stackoverflow.com/a/61481631/1218712

In this case, I think using `??` is better since. If `coordinatesPrecision` is 0, then `||` would consider it to be falsy and it would take the default value.